.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+  push:
+    branches:
+      - master
+
+env:
+  NX_CLOUD_TOKEN: ${{ secrets.NX_CLOUD_TOKEN }}
+  NX_BRANCH: ${{ github.head_ref }}
+  NX_FORCE_ALL: ${{ fromJSON('["", "--all"]')[ inputs.nx-force-all ] }}
```

Comment:
as you don't use this and we have no inputs, I'm assuming this can be removed. What do you think?
```suggestion
```